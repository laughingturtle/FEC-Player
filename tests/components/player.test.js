import React from 'react';
import styled from 'styled-components';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
// import '../../client/components/controls.css';

import Player from '../../client/components/player.jsx';

// describe('<Player />', () => {
//   describe('render()', () => {
//   test('renders the component', () => {
//     const wrapper = shallow(<Player />);
//     expect(wrapper).toMatchSnapshot();
//     })
//   })
// });
// describe('Player', () => {
//   test('it renders', () => {
//     const wrapper = shallow(<Player />)
//     expect(wrapper).toMatchSnapshot();
//     // expect(true).toBeTruthy();
//   })
// })

test('Player renders', () => {
  const wrapper = shallow(<Player />)
  expect(wrapper).toMatchSnapshot();
})