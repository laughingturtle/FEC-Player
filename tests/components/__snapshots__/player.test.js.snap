// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player renders 1`] = `
<styled.section>
  <styled.div>
    <br />
    <styled.div />
  </styled.div>
  <Styled(H5AudioPlayer)
    autoPlay={true}
    src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/557257/wwy.mp3"
  />
</styled.section>
`;
